index.html

```diff
@@ -6,94 +6,223 @@
+                  />
+                </svg>
+              </a>
+            </li>
+            <li class="left-margin-for-icon">
+              <a href="https://twitter.com/CodingC31597" title="twitter"
+                ><svg
+                  width="20"
+                  height="20"
+                  viewBox="0 0 20 20"
+                  fill="none"
+                  xmlns="http://www.w3.org/2000/svg"
+                >
+                  <path
+                    d="M19.7587 0.394423C19.4253 -0.00424693 18.8587 -0.103914 18.4253 0.195088L16.492 1.52399C15.592 0.560536 14.3253 0.0289756 12.9587 0.0289756C10.2587 0.0289756 8.02532 2.22166 8.02532 4.94591C8.02532 5.34458 8.05865 5.71002 8.15865 6.07547C5.82532 5.57713 4.42532 3.88279 3.89198 0.826316C3.82532 0.427646 3.52532 0.128643 3.12532 0.0289756C2.75865 -0.070692 2.35865 0.0954206 2.12532 0.394423C2.05865 0.460868 -3.44135 8.10205 4.52532 14.5804C3.75865 16.2083 1.45865 17.6037 0.558651 18.0688C0.125318 18.2681 -0.0746825 18.7332 0.0253175 19.1984C0.125318 19.6635 0.525318 19.9625 0.991984 19.9625H2.72532C12.8587 19.9625 17.4587 14.9459 17.5587 3.71667L19.6587 1.72332C20.0587 1.3911 20.092 0.793093 19.7587 0.394423Z"
+                  />
+                </svg>
+              </a>
+            </li>
+          </ul>
+        </header>
+      </section>
+
+      <section>
+        <div class="my-recent-work">
+          <h1 class="title">My Recent Project</h1>
+
+          <div class="stroke"></div>
+          <div>
+            <img
+              class="image-placeholder"
+              src="./image/placeholder.png"
+              title="image-placeholder"
+              alt=""
+              srcset=""
+            />
+          </div>
+          <h1 class="title-post">Multi-post stories</h1>
```

Comment:
Right here too there can never be 2 <h1> in your file, and if you check your file from the top you will figure out that you have already use the <h1>, there can only be 1 in your HTML file, please see to this and everywhere you did something similar